pyfar/dsp/filter/audiofilter.py

```diff
@@ -349,6 +480,110 @@ def high_shelve_cascade(
         signal, frequency, frequency_type="lower", gain=None, slope=None,
         bandwidth=None, N=None, sampling_rate=None):
     """
+    Create and apply constant slope filter from cascaded 2nd order high shelfs.
+
+    The filters - also known as High-Schultz filters (cf. [#]_) - are defined
+    by their characteristic frequency, gain, slope, and bandwidth. Two out of
+    the three parameter `gain`, `slope`, and `bandwidth` must be specified,
+    while the third parameter is calculated as
+
+    ``gain = bandwidth * slope``
+
+    ``bandwidth = abs(gain/slope)``
+
+    ``slope = gain/bandwidth``
+
+    :py:func:`~pyfar.dsp.filter.high_shelve_cascade` will be deprecated in
```

Comment:
A general note, no idea what is more convenient, in Coordinates we always put this note at the very beginning.

Reply:
Probably better to put it at the beginning